paperqa/prompts.py

```diff
@@ -77,6 +77,21 @@
 where `summary` is relevant information from text - {summary_length} words and `relevance_score` is the relevance of `summary` to answer question (out of 10).
 """  # noqa: E501
 
+agent_system_prompt = (
+    # Matching https://github.com/langchain-ai/langchain/blob/langchain%3D%3D0.2.3/libs/langchain/langchain/agents/openai_functions_agent/base.py#L213-L215
+    "Optional system prompt message to precede the below agent_prompt."
```

Comment:
I pasted the wrong value here, it should have been:

> You are a helpful AI assistant.

This was fixed in https://github.com/Future-House/paper-qa/pull/683